source/components/switch.template.html

```diff
@@ -0,0 +1 @@
+<script>document.location = '/components/switch.template/';</script>
```

Comment:
No need to add this file, it's only for backwards compatibility for components that existed before we migrated jekyll version